tests/media/test_ffmpeg_inline_encode_decode.liq

```diff
@@ -74,7 +80,9 @@ output.file(
   s
 )
 
-s = ffmpeg.decode.audio_video(s)
+s = ffmpeg.decode.audio_video(id="decoder", s)
+
+clock.assign_new(sync='none',[s])
```

Comment:
```suggestion
clock(sync='none', s)
```

Reply:
This operator is actually giving me trouble. It. returns a new source, which needs to be properly hooked-up. Any reason not to use `clock.assign_new`?

Reply:
Only that it is shorter :) But you can forget about it!